README.md

```diff
@@ -51,6 +51,6 @@ docker push chennaitricolor/wastexchange-fe:latest
 	```
 	ansible-playbook -i deployment/inventory.yaml \
 		--private-key <ssh-private-key> \
-		--extra-vars "env=<staging|prod> app_version=<docker-image-tag>" \
```

Comment:
`app_version=<docker-image-tag>"` looks more intuitive than `$(git rev-parse --short HEAD)`. what do you think?

Reply:
My pref is to make it as non-interactive as possible. By that I mean, the number of items that the human has to parse, understand and replace in the above command-line should be as minimal as possible.

Reply:
Agreed. I was asking that because I felt it's easy to make a mistake with `$(git rev-parse --short)`. 

It should be fine if whoever executes it is aware of what it's doing.

Reply:
@arunvelsriram - the commands to build and publish the tagged images also use the same command snippet. Please approve the changes.